### DISCUSSION TITLE
Filter relevant items in "for you" feed

### DISCUSSION THREAD

**Discussion by urld** [community/community#14040] [Other Feature Feedback, Questions, & Ideas]
While some items in the new feed are helpful (e.g. new releases), others are purely spam to me (random people starring random repos).
I guess everyone has a different perception on "helpful" or "important", so maybe it would be best to allow to select which type of items i font to see in my feed

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
